src/collectors/fluentd/fluentd.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+
+"""
+The FlutendCollector monitors fluentd and data about the kinesis stream
+
+#### Dependencies
+
+ * flutend
+ * requests
+
+#### Example config
+
+```
+    enabled = True
+    host = localhost
+    port = 24220
+    [[[collect]]]
+        kinesis = buffer_queue_length, buffer_total_queued_size, retry_count
+```
+
+"""
+
+import diamond.collector
+import requests
```

Comment:
I am not sure, but I feel like we generally don't add dependencies on things like `requests` when the built-in libraries will do. It's a little less convenient for the author for sure, but it's *much* more convenient for the many future users of this collector. A quick grep of the entire diamond source tree seems to indicate that `import requests` appears in no other places, plausibly for this exact reason.